cloudbuild.yaml

```diff
@@ -0,0 +1,68 @@
+timeout: 3600s # 1 hour
+logsBucket: 'gs://oppia-cloudbuild-logs'
+steps:
+  - id: build-oppia-base
+    name: gcr.io/kaniko-project/executor:latest
```

Comment:
Is this your container? I'm asking so we know who to reach out to with questions

Reply:
This is one of the official cloud build containers, so it's maintained by Google. Here's more info: 
https://cloud.google.com/cloud-build/docs/kaniko-cache#kaniko-build